### DISCUSSION TITLE
Mesh networking

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [stoatchat/discussions#349] [Q&A] [answered]
As a game voice communication, I will recommend it to my friends revolt.
As a social chat, I would never recommend revolt to my friends.

Reasons
revolt is not used [Mesh networking](https://en.wikipedia.org/wiki/Mesh_networking)
As soon as the country blocked revolt he wouldn't use the VPN and I lost my friend permanently.
CENSORSHIP-RESISTANT
With no central authority controlling the network, content hosted over Mesh networking  [yggdrasil](https://yggdrasil-network.github.io/) and  [nebula](https://github.com/slackhq/nebula) WebRTC p2p can’t be censored.

**Comment by sussycatgirl** [comment, marked as answer]
Mesh networking has nothing to do with Revolt. I think you mean federation, which won't be happening: https://developers.revolt.chat/faq/federation
If you're suffering from government censorship, Revolt might not be the best choice for you.
